tests/test_server.py

```diff
@@ -1,12 +1,15 @@
+import pytest
+
 from dynagrpc import GrpcServer, GrpcTestClient
 
 
-def test_add():
+@pytest.mark.parametrize("first,second,expected", [(5, 7, 12), (6, -9, -3), (-1, -3, -4)])
```

Comment:
This line is quite big. We're not enforcing any linter yet, but I think it would be reasonable to keep PEP-8 and its standard 79 line length (which quite is helpful for vertical splitting workflows).

Reply:
The commits problem was solved. I performed a rebase and fixed the problem.
And also, I solved the issue of the long line.